microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -747,20 +747,38 @@ To enable diagnostic data collection, go to `edge://settings/privacy`. Turn on t
 <!-- ====================================================================== -->
 ## Writing assistance
 
-Microsoft Edge provides enhanced spelling and grammar checking as you type in the browser.  To use enhanced spelling and grammar checking, go to `edge://settings/languages` and select **Microsoft Editor** under the **Use writing assistance** toggle. 
+To help you write faster and with fewer mistakes on the web, Microsoft Edge provides different writing assistance tools, including enhanced spelling and grammar checking and text prediction. 
```

Comment:
Do we need to have the word different.  I think we could eliminate it.

Reply:
The word 'different' was removed in this PR at some point.
Latest:
https://github.com/MicrosoftDocs/edge-developer/pull/2291/files

Please close this comment thread.  @JasonAndrewWriter